Assignment_1/Assignment_1/main.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  main.swift
+//  Assignment_1
+//
+//  Created by Brendon Cecilio on 10/1/19.
+//  Copyright © 2019 Brendon Cecilio. All rights reserved.
+//
+
+import Foundation
+import GameplayKit
+
+print("Welcome to my knock off Dungeons and Dragons Game 🐲!")
+print()
+sleep(4)
+
+print("Choose your weapon! (Axe/Staff/Bow and Arrow)")
+let weapon = readLine()?.lowercased() ?? ""
```

Comment:
great job in lowercasing the user's response so there is less human error and guarding against nils using nil coalescing but you are giving them a blank default as a weapon which your code can't use. So you would want to give them a "staff", "axe", or "bow and arrow" to use as default so it can be used in your code properly